### ISSUE TITLE
haskellPackages.implicit-hie fails to build (test phase)

### ISSUE THREAD

**Issue by turion** [NixOS/nixpkgs#91505]
**Describe the bug**
The haskellPackages.implicit-hie test phase fails.

**To Reproduce**

```
nix-repl> :b haskellPackages.implicit-hie
builder for '/nix/store/h08h7ln5ykxfha6rci51yhybm7kpjklf-implicit-hie-0.1.0.0.drv' failed with exit code 1; last 10 log lines:
  
    To rerun use: --match "/Should Succeed/successfully generates cabal hie.yaml for haskell-language-server/"
  
  Randomized with seed 1714634695
  
  Finished in 0.0026 seconds
  14 examples, 3 failures
  Test suite implicit-hie-test: FAIL
  Test suite logged to: dist/test/implicit-hie-0.1.0.0-implicit-hie-test.log
  0 of 1 test suites (0 of 1 test cases) passed.
[0 built (1 failed)]
error: build of '/nix/store/h08h7ln5ykxfha6rci51yhybm7kpjklf-implicit-hie-0.1.0.0.drv' failed
```

Disabling the test phase fixes the issue:
```
nix-repl> :b haskell.lib.dontCheck haskellPackages.implicit-hie

this derivation produced the following outputs:
  doc -> /nix/store/3bm8w9j42314a2k5rb99hwqm85pksrsy-implicit-hie-0.1.0.0-doc
  out -> /nix/store/b5rvbgjs14c35ix68vdwd4sr000mpsaa-implicit-hie-0.1.0.0
```

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.47, NixOS, 20.09pre231796.22a81aa5fc1 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"musnix, nixos-20.09pre231796.22a81aa5fc1, nixos-hardware"`
 - channels(turion): `"home-manager, nixos-19.03-19.03.173691.34c7eb7545d, nixos-unstable-newer-20.09pre231796.22a81aa5fc1, nixpkgs-20.09pre231837.2cd2e7267e5"`
 - nixpkgs: `/home/turion/.nix-defexpr/channels/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - haskellPackages.implicit-hie
# a list of nixos modules affected by the problem
module:
```

**Comment by svmhdvn** [comment]
This seems to build on `haskell-packages` with a successful test phase, this issue can be closed.

**Comment by turion** [comment]
@svmhdvn Can you link to the relevant PR?

**Comment by svmhdvn** [comment]
A quick git bisect shows that this commit fixes it:
https://github.com/NixOS/nixpkgs/commit/af9608d6d133ad9b6de712418db52603bbc8531c

Upstream, the fixes were made to the tests for release 0.1.1.0:
https://github.com/Avi-D-coder/implicit-hie/compare/0.1.0.0...0.1.1.0

**Comment by turion** [comment]
Awesome, thanks!
